### ISSUE TITLE
_LOADIMAGE and Relative File Paths

### ISSUE THREAD

**Issue by ProxyPlayerHD** [Galleondragon/qb64#106]
### TL;DR:
Relative file paths are starting from the QB64 directory instead of the directory that the .BAS file (that is currently open in QB64 or being compiled) is located.
and i'm 99% sure it's not intended, i hope so atleast.

### Long version:

A strange thing i found while trying to program a small game.

for context the .BAS file of the program is located in some directory, in the same directory there is another folder called "texture" which contains all the textures used in the program.

Problem is, you cannot use Relative paths to load images from that folder, they either have to be in the same directory as the .BAS file
(so the path would just be "\<filename>.\<extension>")
or you have to use an Absolute path.
(so the path would be something like "C:\Folders\Folders\ \<filename>.\<extension>")

I only found this out becuase i made a new program to experiment with the _LOADIMAGE keyword. i made a test folder in the BQ64 directory, put some image in it, and it worked perfectly fine with a relative path.

then it hit me... whenever you use a relative path it doesn't start where the .BAS file is located, but it starts from the QB64 directory.

i tested that idea by copying the "texture" folder from the program i was working on to the QB64 directory, compiled the program, and it worked.

**Comment by SteveMcNeill** [comment]
File paths are relative to where the EXE is located.  Select the option to,”Save EXE in source directory”.

**Comment by FellippeHeitor** [comment]
![84F75CFA-9D0B-477B-AA51-AB6D3E18631D](https://user-images.githubusercontent.com/10699359/69687426-a3140a00-10a1-11ea-85ec-cee213a4f128.jpeg)

**Comment by ProxyPlayerHD** [comment]
but it does that by default...
so i didn't notice the file not being saved in the correct folder.
